lib/pacemaker/pcmk_sched_group.c

```diff
@@ -149,36 +149,44 @@ group_create_actions(pe_resource_t * rsc, pe_working_set_t * data_set)
     for (; gIter != NULL; gIter = gIter->next) {
         pe_resource_t *child_rsc = (pe_resource_t *) gIter->data;
 
-        child_rsc->cmds->create_actions(child_rsc, data_set);
+        child_rsc->cmds->create_actions(child_rsc);
         group_update_pseudo_status(rsc, child_rsc);
     }
 
     op = start_action(rsc, NULL, TRUE /* !group_data->child_starting */ );
```

Comment:
For a future commit, it'd be nice to make the value passed just be the thing that's commented out.

Reply:
child_starting (and this comment) is actually removed in a later PR